shell/agents/Microsoft.Azure.Agent/Schema.cs

```diff
@@ -386,7 +387,8 @@ internal async Task CreateOrRenewTokenAsync(CancellationToken cancellationToken)
 
             if (needRefresh)
             {
-                _accessToken = await new AzureCliCredential()
+                _accessToken = await new ChainedTokenCredential(new AzureCliCredential(),
+                    new InteractiveBrowserCredential(ApplicationId))
```

Comment:
There is a good idea, but we will need to do a lot more:
1. What about scenarios where browser is not available? Like when user is using AIShell on a remote machine via SSH connection. How do we detect that and switch to "device code" login?
2. The error handling (e.g. error messages to help with mitigation) regarding access token failures will need to be updated accordingly.
3. Maybe we should not recommend `/replace` when AzCLI is not installed. And we shouldn't recommend `/code post` for AzCLI commands when it's not installed.